prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -188,6 +192,7 @@ def build_agent(
     api_base: str = "https://api.openai.com/v1",
     long_term_memory: LongTermMemoryTableHandler | None = None,
     import_actions_from_memory: int = 0,
+    max_tokens: int = 8196,
```

Comment:
Previously default value was 512, but our current system prompt is ~900 tokens, so 512 is too little for the agent to write its own system prompt.